relational_database/homework.py

```diff
@@ -150,7 +181,9 @@ def task_12_list_suppliers_from_specified_countries(cur):
 
     Returns: 8 records
     """
-    pass
+    cur.execute("""SELECT * FROM suppliers 
+                   WHERE country = 'USA'  AND country='UK' AND country='Japan';""")
```

Comment:
You might use operator IN ('USA', 'UK' ....)

Reply:
OK, done. added specified fields in select and changed operator where.